packages/calypso-e2e/src/lib/pages/marketing-page.ts

```diff
@@ -11,13 +11,14 @@ import { Page } from 'playwright';
 
 const selectors = {
 	content: '#primary',
-	navtabsList: '.section-nav-tabs__list',
+	navTabs: 'div.section-nav-tabs',
+	navTabsDropdownOptions: '.select-dropdown__option',
```

Comment:
Our navtabs become a non-standard dropdown when the viewport is small enough.

This also means that in `clickTabItem`, it isn't possible to simply use Playwright's built-in `selectOption` call.